### PR TITLE
Add Navbar desktop and Mobile UI with react router

### PR DESCRIPTION
**Pull request by AamirAlam**
## Changes
- Add new navbar UI as per the mock
- Handled Navbar for mobile screens

- Before 
navbar_bwef

- After Desktop screen
navbar

- Mobile screen
navbar-mob

- Internationalization support

internationalization

### CODE REVIEW

File: src/component/layout/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+    display: "flex",
+    [theme.breakpoints.up("md")]: {
+      display: "none",
+    },
+  },
+}));
+
+const Navbar = () => {
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = React.useState(null);
```
Comment (ArfatSalman) [inline src/component/layout/Navbar.js:88]: Shouldn't you be using `useRef` as opposed to `useState` or something?

**Review by ArfatSalman — COMMENTED** [review]
Looks good. Waiting for the i18n support!

File: src/component/layout/Navbar.js (hunk 1 above)
Comment (AamirAlam) [reply src/component/layout/Navbar.js:88]: Okay

Comment (AamirAlam) [reply src/component/layout/Navbar.js:88]: I'll add the i18th setup in next commit

**Pull request merged**